src/models/m_rsc.erl

```diff
@@ -235,7 +236,7 @@ get(Id, Context) ->
     end.
 
 %% @doc Get the resource from the database, do not fetch the pivot fields.
--spec get_raw(resource(), #context{}) -> list().
+-spec get_raw(resource(), #context{}) -> props().
```

Comment:
The spec in line 239 uses a record, please define a type for the record and use that instead.